Don't create bare spacing utilities with invalid multiples (#14962)
tailwindlabs/tailwindcss: CHANGELOG.md

### Fixed

- Don't reset horizontal padding on date/time pseudo-elements ([#14959](https://github.com/tailwindlabs/tailwindcss/pull/14959))

> - Don't emit `calc()` with invalid values for some negative utilities that use the spacing multiplier ([#14962](https://github.com/tailwindlabs/tailwindcss/pull/14962))

Reviewer: ```suggestion
- Don't emit `calc()` with invalid values for bare values that aren't integers in spacing utilities ([#14962](https://github.com/tailwindlabs/tailwindcss/pull/14962))
```
